### PR TITLE
Fix Open in Portal command

### PR DESCRIPTION
**Pull request by motm32**

### CODE REVIEW

File: src/services/AzureSubscriptionProvider.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
 *  Licensed under the MIT License. See License.md in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 import * as arm from '@azure/arm-subscriptions';
-import type { Environment } from '@azure/ms-rest-azure-env';
+import { Environment } from '@azure/ms-rest-azure-env';
```
Comment (bwateratmsft) [inline src/services/AzureSubscriptionProvider.ts:6]: This package will need to be added as an explicit dependency in package.json

File: src/services/AzureSubscriptionProvider.ts [hunk 2]
```diff
@@ -184,7 +184,7 @@ export class VSCodeAzureSubscriptionProvider extends vscode.Disposable implement
                     authentication: {
                         getSession: () => session
                     },
-                    environment: {} as Environment,
+                    environment: Environment.AzureCloud,
```
Comment (alexweininger) [inline src/services/AzureSubscriptionProvider.ts:187]: Can we hardcode this to `AzureCloud` or do we need to support other environments?

Comment (motm32) [reply src/services/AzureSubscriptionProvider.ts:187]: I think we're only trying to support `AzureCloud` at the moment.

Comment (alexweininger) [reply src/services/AzureSubscriptionProvider.ts:187]: Ok, that's fine. Are we planning to add support for the other clouds later?

**Review by bwateratmsft — APPROVED** [review]
Code looks fine to me but please work out the sovereign/custom cloud question before merging :smile:

File: src/services/AzureSubscriptionProvider.ts (hunk 2 above)
Comment (motm32) [reply src/services/AzureSubscriptionProvider.ts:187]: For the first iteration were not planning to support any other clouds, but as we migrate the vs code auth provider we may add support for sovereign/custom cloud.

**Pull request merged**